### PR TITLE
refactor(layouts): create story and update style for base layout

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description
Create a story for `RootLayout`, now named `BaseLayout` to be in sync with naming in the DS.

Also makes slight layout adjustments to adhere to the DS.

### CODE REVIEW

File: src/layouts/layoutStories/BaseLayout.stories.tsx
Comment (pettinarip) [inline src/layouts/layoutStories/BaseLayout.stories.tsx:1]: shouldn't the folder be named just `stories`? to avoid the redundancy.

**Review by pettinarip — COMMENTED** [review]
Nice to see a story for this!!

File: src/layouts/layoutStories/BaseLayout.stories.tsx
Comment (TylerAPfledderer) [reply src/layouts/layoutStories/BaseLayout.stories.tsx:1]: No problem!

File: src/layouts/BaseLayout.tsx
Comment (TylerAPfledderer) [reply src/layouts/BaseLayout.tsx:54]: @pettinarip Hmm that token is set to `1440px` which is not the width shown in figma. Should we increase the value to `1536px` or add `container.3xl`?

Comment (pettinarip) [reply src/layouts/BaseLayout.tsx]: Hmmm what the DS says about it?

I think it is the moment to update these values to this
image

We are using `container.2xl` in the header, so it should match the layout width.

Comment (TylerAPfledderer) [reply src/layouts/BaseLayout.tsx]: Oh, I didn't even see that! haha ok I will update that here.

Comment (TylerAPfledderer) [reply src/layouts/BaseLayout.tsx]: This update has now been pushed.

**Comment by TylerAPfledderer** [comment]
I am aware of the current conflict with the old `RootLayout` in prod

**Comment by wackerow** [comment]
We refactored where we convert this Date into a readable string, and we're now passing `lastDeployLocaleTimestamp` instead of `lastDeployDate`. Updated the two instances of this variable name which fixed the build issue locally for me.

**Review by wackerow — APPROVED** [review]
Lgtm!

**Comment by wackerow** [comment]
Seeing 113 items in the Chromatic change set. These look good to me, but @TylerAPfledderer should @nloureiro sign off on these before pulling in?

**Comment by TylerAPfledderer** [comment]
> Seeing 113 items in the Chromatic change set. These look good to me, but @TylerAPfledderer should @nloureiro sign off on these before pulling in?

We are trying to leave them to review during staging

**Comment by nloureiro** [comment]
yes, I went through all of the differences, and they seem ok. approved on chromatic

**Comment by pettinarip** [comment]
> > Seeing 113 items in the Chromatic change set. These look good to me, but @TylerAPfledderer should @nloureiro sign off on these before pulling in?
> 
> We are trying to leave them to review during staging

Is that necessary? maybe Im not following you on what you mean with staging but shouldn't they all appear when we create the PR pointing to staging?

I think we should review & fix them if we are adding them here since that is the scope of the PR, right?

**Comment by TylerAPfledderer** [comment]
> > > Seeing 113 items in the Chromatic change set. These look good to me, but @TylerAPfledderer should @nloureiro sign off on these before pulling in?
> > 
> > We are trying to leave them to review during staging
> 
> Is that necessary? maybe Im not following you on what you mean with staging but shouldn't they all appear when we create the PR pointing to staging?
> 
> I think we should review & fix them if we are adding them here since that is the scope of the PR, right?

I recall having the discussion previously during the QA sessions. So that Nuno is not tasked to look over every single PR that had a change, we leave the UI Review check open for Nuno to view changeset during the QA Session days when the deploy PR is created. As the final "signoff" of sorts.

If it appears that might not also be ideal, then should that check block these PRs until Nuno approves it?

**Review by corwintines — APPROVED** [review]
LGTM!

**Pull request merged**
